Makefile

```diff
@@ -1,8 +1,50 @@
+# List of folders containing Semgrep rule files (with a .yml suffix).
+#
+# This is meant to exclude folders that don't contain rule files and
+# may contain .yml files that are not Semgrep rules and would result
+# in errors.
+#
+RULE_FOLDERS = $(shell ls -d */ | grep -v '^stats')
```

Comment:
I would also exclude `trusted_python` and `fingerprints` folders
the first one does not contain rules
second one contains CAI rules that do not have tests

Reply:
oh `stats` and `fingerprints` are removed before the scan, so no worries about them
https://github.com/returntocorp/semgrep-rules/actions/runs/3019231154/workflow#L22

Reply:
thanks for taking a look, I hadn't noticed that these folders were being removed in the CI job. In semgrep, we used to remove `stats/` but it's unclean. Here's what I'm going to do:
1. filter out `trusted_python` and `fingerprints` in addition to `stats` in the makefile
2. make the CI job use the makefile. This will be https://github.com/returntocorp/semgrep-rules/issues/2380 which right now waiting on semgrep becoming available (and also, git permission issues).

Reply:
(1) is done. (2) will be done later.